### ISSUE TITLE
Episode00-Before We Start - More Basic step-by-step setup instructions

### ISSUE THREAD

**Issue by raqmann** [datacarpentry/python-ecology-lesson#276]
Need more step-by-step instructions for setup, including sample workflow and screenshots. For a very beginner level, the setup is very text heavy. Screenshots with numbered steps would be so helpful. 

Examples: 

-Organizing Working Directory: not clear where to create directories, 
-Difficult for new learners to understand how Python, Pandas, Anaconda work together in setup. What will they see when they download packages? How do they navigate each tool? What should they look like on screen?

**Comment by btovar** [comment]
Closing in favor of #457
